### DISCUSSION TITLE
Winver (Windows Version)

### DISCUSSION THREAD

**Discussion by Pixel556** [DustinBrett/daedalOS#139] [Feature Requests]
Can you atleast make the winver? The only changes you should do: Change the Windows logo icon to the website favicon, change the colour of the "Windows 10" text black, replace "Windows 10" text with "daedalOS", and change the (OS Build XXXXXXX.XXX) to (Version*insert version here*)
![image](https://user-images.githubusercontent.com/95143545/162610741-a9680673-76d0-4a0d-98ab-4ff7102b6dda.png)

**Comment by Pixel556** [comment]
Add it as a start menu shortcut, and name it Windows Version

**Comment by DustinBrett** [comment]
Thanks for the suggestion. I plan at some point to add more version info indeed.

**Comment by Pixel556** [comment]
This is an old screenshot. I have now downgraded to Windows 10.

**Comment by DustinBrett** [comment]
Here is my winver.exe

![image](https://user-images.githubusercontent.com/16656318/176259316-67b42b84-24ba-4d54-9474-8ec4defd81f7.png)

**Reply by Pixel556** [reply]
We both have the same version.
![image](https://user-images.githubusercontent.com/95143545/177772499-19702612-fdae-4095-9fc4-93b6a17a589f.png)

**Comment by DustinBrett** [comment]
Nice! Ya 21H2 is the latest release build. I keep up to date so that I can make sure my site looks like the latest Windows 10 build.
